dap_job_quality/utils/spacy_keyword_search.py

```diff
@@ -0,0 +1,105 @@
+from dap_job_quality.getters.ojo_getters import get_ojo_sample
+from dap_job_quality.utils.text_cleaning import clean_text
+
+
+import pandas as pd
+import spacy
+from spacy.matcher import PhraseMatcher, Matcher
+import spacy
+from spacy import displacy
+from spacy.matcher import Matcher
+
```

Comment:
You could maybe load a model here ie
```
nlp = spacy.load("en_core_web_sm")
```
And set that as the default `nlp` argument for these functions?

Reply:
Nice! :)